From 22e02998dec8af15a25b7e8e3d53c174810cc3be Mon Sep 17 00:00:00 2001
From: Dag Andersen <danders@get2net.dk>
Date: Thu, 11 Jun 2020 10:43:38 +0200
Subject: [PATCH] Make show hidden row(s) work

Check for hidden rows was inverted to check for shown rows,
hence hidden rows would never be unhidden.

BUG:338816
BUG:354236
FIXED-IN:3.2.2
(cherry picked from commit 28920c50e6300d643477f256aba91c8c51836682)
---
 sheets/commands/RowColumnManipulators.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sheets/commands/RowColumnManipulators.cpp b/sheets/commands/RowColumnManipulators.cpp
index 95282d6e31b..25e8c454e27 100644
--- a/sheets/commands/RowColumnManipulators.cpp
+++ b/sheets/commands/RowColumnManipulators.cpp
@@ -192,7 +192,7 @@ bool HideShowManipulator::preProcessing()
                     }
                 }
                 for (int row = range.top(); row <= range.bottom(); ++row) {
-                    if (!m_sheet->rowFormats()->isHidden(row)) {
+                    if (m_sheet->rowFormats()->isHidden(row)) {
                         region.add(QRect(1, row, KS_colMax, 1));
                     }
                 }
